added page and logo for Warmup thermostat (#9396)
home-assistant/home-assistant.github.io: source/_components/climate.warmup.markdown

---
layout: page
title: "Warmup Thermostat"
description: "Instructions on how to integrate Warmup (4IE) thermostats within Home Assistant."
date: 2019-02-27 12:01
sidebar: true
comments: false
sharing: true
footer: true
logo: warmup.png
ha_category: Climate

> ha_release: 0.93

Reviewer: ```suggestion
ha_release: 0.94
```

ha_iot_class: Cloud Polling

> ha_qa_scale:

Reviewer 2: If there is no scale, this should be removed.

---

> The `warmup` climate platform let you control Warmup 4IE thermostats from Home Assistant.

Reviewer 2: ```suggestion
The `warmup` climate platform lets you control Warmup 4IE thermostats from Home Assistant.
```

To set it up, add the following information to your `configuration.yaml` file:

```yaml
climate:
  - platform: warmup
    username: YOUR_E_MAIL_ADDRESS
    password: YOUR_PASSWORD
    location: YOUR_LOCATION_NAME
    room: YOUR_ROOM_NAME
```

<p class='note'>

> To use this component, you have to setup an account at the warmup web site first and register your thermostat appropriately. See <https://my.warmup.com/login>.

Reviewer 2: ```suggestion
To use this component, you have to set up an account at the warmup web site first and register your thermostat appropriately. See <https://my.warmup.com/login>.
```

</p>

{% configuration %}
name:

>   description: Name of thermostat.

Reviewer 2: ```suggestion
  description: Name of the thermostat.
```

  required: false
  default: warmup4ie
  type: string
username:
  description: The username for accessing your Warmup account.
  required: true
  type: string
password:
  description: The password for accessing your Warmup account.
  required: true
  type: string
location:
  description: The name of the location from your Warmup account.
  required: true
  type: string
room:
  description: The name of the room from your Warmup account.
  required: true
  type: string

> {% endconfiguration %}

Reviewer 2: ```suggestion
{% endconfiguration %}

```
